recipe/meta.yaml

```
@@ -16,26 +16,31 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
-  run_constrained:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+  host:
+    - zlib
+    - bzip2
 
 test:
   source_files:
     - source/test
   requires:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}
```

```suggestion
    - {{ stdlib("c") }}
    - {{ compiler('cxx') }}
    - {{ compiler('fortran') }}
    - coreutils  # [osx]
```

We can try to see if helps, but we might need to update homebrew for `osx_64`.

---

If this suggested change doesn't work then you can try adding `brew install coreutils` as the first test command.

```yaml
  commands:
    - # Ensure 'g' prefixed Unix commands available for macOS
    - brew install coreutils  # [osx]

    # Check HepMC3-config values
    - HepMC3-config --prefix
```

If it _does_ work then `coreutils  # [osx]` should get added to the `run` requirements.

---

@Kwaizer Never mind. I checked the conda-forge `coreutils` and it doesn't have the 'g' prefixed commands. So this suggestion won't work.

cc @henryiii and @chrisburr for ideas here on how to deal with macOS expecting the `g` prefixed `coreutils` commands.